bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -73,6 +74,9 @@ class App extends Component {
     document.getElementsByTagName('html')[0].style.fontSize = this.props.fontSize;
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    console.log(this.state.width);
```

Comment:
why that?

Reply:
these lines are gone already: https://github.com/bigbluebutton/bigbluebutton/pull/5442/commits/cafce78b0f54d5d83f2741e99314bea8419201c7#diff-31de147950b401db04faf10d366f7af7L77